services/AsigurareService.java

```diff
@@ -0,0 +1,41 @@
+package services;
+
+import entities.Asigurare;
+import interfaces.AsigurareInterface;
+
+public class AsigurareService implements AsigurareInterface {
```

Comment:
[Fix] toate serviciile tale nu fac decat sa apeleze getteri si setteri, nu ai nimic logica de business si nici metode de interactiuni cu baza de date, de tipul CRUD

Reply:
Oscilam intre: utilizatorul sa nu foloseasca niciodata getteri si setteri si sa se foloseasca doar de servicii pentru a manipula datele, inclusiv apelarea de getteri din servicii, si intre sa las in servicii doar logica de business. Inteleg acum ca a doua varianta e cea potrivita.